hypothesis-python/src/hypothesis/internal/reflection.py

```diff
@@ -436,7 +436,7 @@ def extract_lambda_source(f):
     return source
 
 
-def get_pretty_function_description(f):
+def get_pretty_function_description(f: Any) -> str:
```

Comment:
```suggestion
def get_pretty_function_description(f: object) -> str:
```

Reply:
not technically true, since not all objects have `__name__`. this is fine dynamically because we gate behind a hasattr, but we do call the function with non-objects and mypyc will enforce this (as an error) at runtime if we type check as object

Reply:
I'd expect switching to `getattr` with a sentinel value would work?  But it's not a big deal either way.